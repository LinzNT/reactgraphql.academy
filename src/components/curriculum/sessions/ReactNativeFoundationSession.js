import React from 'react'
import Ul, { Li } from '../../layout/Ul'
import Session from './Session'

const ReactNativeFoundationSession = ({ title }) => (
  <Session title={title}>
    <Ul>
      <Li>Architecture</Li>
      <Li>Project setup</Li>
      <Li>Layout</Li>
      <Li>Styling</Li>
      <Li>Navigations with react-navigation</Li>
    </Ul>
  </Session>
)

export default ReactNativeFoundationSession
