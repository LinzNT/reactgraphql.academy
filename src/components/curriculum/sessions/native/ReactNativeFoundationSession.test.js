import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import ReactNativeFoundationSession from './ReactNativeFoundationSession'

describe('<ReactNativeFoundationSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<ReactNativeFoundationSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
