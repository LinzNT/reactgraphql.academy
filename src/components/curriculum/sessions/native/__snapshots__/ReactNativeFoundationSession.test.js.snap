// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactNativeFoundationSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Architecture
    </styled.li>
    <styled.li>
      Project setup
    </styled.li>
    <styled.li>
      Layout
    </styled.li>
    <styled.li>
      Styling
    </styled.li>
    <styled.li>
      Forms
    </styled.li>
  </styled.ul>
</Session>
`;
